apps/backend/src/utils/__tests__/logger.test.ts

```
@@ -0,0 +1,141 @@
+      const metadata = { userId: 'demo', action: 'test' };
+      logger.info('test message', metadata);
+
+      expect(consoleSpy.log).toHaveBeenCalledWith(
+        expect.stringContaining(JSON.stringify(metadata))
+      );
+    });
+
+    it('should set appropriate log level based on NODE_ENV', () => {
+      // Test development environment
+      process.env.NODE_ENV = 'development';
+      delete process.env.LOG_LEVEL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)